.ci/bump-releasenotes-builder-version.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+set -e
+
+POM_FILE=releasenotes-builder/pom.xml
+
+MAVEN_METADATA=https://repo1.maven.org/maven2/com/puppycrawl/tools/checkstyle/maven-metadata.xml
+LATEST_VERSION=$(curl $MAVEN_METADATA | grep -oPm1 "(?<=<latest>)[^<]+")
+echo LATEST_VERSION="$LATEST_VERSION"
+
+MVN_CURRENT_VERSION=$(grep "^        <checkstyle.version>.*</checkstyle.version>$" "$POM_FILE" | awk -F'[><]' '{print $3}')
+echo MVN_CURRENT_VERSION="$MVN_CURRENT_VERSION"
+
+echo "Checking Version for: "$POM_FILE
+if [ "$MVN_CURRENT_VERSION" != "$LATEST_VERSION"  ]; then
+    cd releasenotes-builder
+    mvn versions:set-property -Dproperty=checkstyle.version -DnewVersion="$LATEST_VERSION"
+    echo "Version updated to $LATEST_VERSION"
+    git config --global user.name 'github-actions[bot]'
```

Comment:
please move commit commands to github action.
Shell scripts should only update target version,
commit should be  single for whole repo. 

in Github Action code you should check if changes exists before doing commit.

I donot see a reason to do MVN_CURRENT_VERSION, just do update on what is take from metadata file, if content is same "git diff" will be empty, so commit will be skipped.

Reply:
1. Commit commands moved to GitHub Action.
2. Shell script code updated to the only updated target version.
3. `MVN_CURRENT_VERSION` removed.
4. GitHub Action commit and condition to check if changes exist code added.